src/components/ListLayout.jsx

```diff
@@ -108,10 +115,17 @@ const ListLayout = ({ items, localToken }) => {
                 <input
                   type="checkbox"
                   checked={within24hours(item.purchasedDate)}
-                  onChange={(e) => handleCheckboxChange(e)}
+                  onChange={(e) => handleCheckboxChange(e, item)}
                   name={item.id}
                   aria-label={item.itemName} //what do we want to call this ('item', 'item.itemName', 'purchased item' .....)
                 />
+                <button
+                  aria-label="delete button"
```

Comment:
My first thought is something like `delete ${item}` might be more clear, but I'd love to hear @EdwardHinkle 's thoughts.

Reply:
Good point - specificity is good!